packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js

```diff
@@ -101,7 +101,7 @@ export class ProductAttributeValue extends PureComponent {
 
     renderBooleanAttribute() {
         const { attribute: { attribute_value } } = this.props;
-        return this.renderStringValue(attribute_value ? __('Yes') : __('No'));
+        return this.renderStringValue(+attribute_value ? __('Yes') : __('No'));
```

Comment:
If you want to convert a value from string to number, better to use `parseInt` of `parseFloat` functions, using `+` in from of a variable is an implicit way of doing this.